### PR TITLE
Fixes #6

### LINKED ISSUE #6: Heroku App - 503 Service error
For some reason using Heroku the service doesn't appear to be responding. I followed the docs. Any insight as to why I'm getting a 503 when testing the POST?

Note, I've changed the URLs so not as to end up with crazy cost from dynos

```
wget --post-data 'test' http://mymattermostinstall.herokuapp.com
--2016-03-25 14:33:24--  http://mymattermostinstall.herokuapp.com/
Resolving mymattermostinstall.herokuapp.com... 50.16.232.151
Connecting to mymattermostinstall.herokuapp.com|50.16.232.151|:80... connected.
HTTP request sent, awaiting response... 503 Service Unavailable
2016-03-25 14:33:24 ERROR 503: Service Unavailable.
```

### PR DESCRIPTION
**Pull request by Lujeni**
- Remove useless part about outgoing webhook
- Slash part use the wrong url, missing the new_post endpoint
- Rewrite the heroku documention using the CLI

### CODE REVIEW

File: HEROKU.md [hunk 1]
```diff
@@ -10,33 +11,25 @@ Here's how to start:
...
- 1. Log in to your Mattermost account. Click the three dot menu at the top of the left-hand side and go to **Account Settings** > **Integrations** > **Outgoing Webhooks**
- 2. Under *Add a new outgoing webhook*, leave the *Channel* unselected and enter `gif:` into **Trigger Words**. You may select a channel if you only want this integration to be available in a specified channel
- 3. Paste your Heroku domain into *Callback URLs*, making sure to add `http://` to the beginning and `/new_post` to the end so it looks similar to `http://myapp.heroku.com/new_post` and click **Add**
- 4. Copy the *Token* from your newly created webhook that appears under the *Existing outgoing webhooks* section
- 5. Go back to your Heroku app dashboard under the *Settings* tab. Under the *Config Variables* section, click **Reveal Config Vars**
- 6. Type `MATTERMOST_GIPHY_TOKEN` as the *KEY* and paste in the token you copied as the *VALUE*, and click **Add**
-
-4. **Connect your project to your Mattermost account for slash commands**
+
+    $ heroku login
```
Comment (ultrabug) [inline HEROKU.md:15]: maybe use ```bash to have a cleaner syntax ?

**Review by ultrabug — CHANGES_REQUESTED** [review]
Thanks, minor formatting changes

File: LINUX.md [hunk 2]
```diff
@@ -18,22 +18,16 @@ Here's how to start:
...
- 2. Under *Add a new outgoing webhook*, leave the *Channel* unselected and enter `gif:` into *Trigger Words*. You may select a channel if you only want this integration to be available in a specified channel
- 3. Paste your Web Server domain into *Callback URLs*, making sure to add `http://` to the beginning and `/new_post` to the end so it looks similar to `http://<your-web-server-domain>/new_post` and click **Add**
- 4. Copy the *Token* from your newly created webhook that appears under the *Existing outgoing webhooks* section
-
-3. **Set up your Mattermost slash command**
+2. **Set up your Mattermost slash command**
  1. Log in to your Mattermost account. Click the three dot menu at the top of the left-hand side and go to **Account Settings** > **Integrations** > **Slash Commands**
  2. Under *Add a new command*, enter `/gif` into **Command Trigger Word**
- 3. Paste your Web Server domain into *Callback URLs*, making sure to add `http://` to the beginning
+ 3. Paste your Web Server domain into *Callback URLs*, making sure to add `http://` to the beginning and `/new_post` to the end so it looks similar to `http://<your-web-server-domain>/new_post` and click **Add**
```
Comment (hanzei) [inline LINUX.md]: I think the port on which the integration is running on musst be include e.g. http://<your-web-server-domain>:5000/new_post

File: LINUX.md [hunk 3]
```diff
@@ -18,22 +18,16 @@ Here's how to start:
...
 
-2. **Set up your Mattermost outgoing webhooks**
- 1. Log in to your Mattermost account. Click the three dot menu at the top of the left-hand side and go to **Account Settings** > **Integrations** > **Outgoing Webhooks**
- 2. Under *Add a new outgoing webhook*, leave the *Channel* unselected and enter `gif:` into *Trigger Words*. You may select a channel if you only want this integration to be available in a specified channel
- 3. Paste your Web Server domain into *Callback URLs*, making sure to add `http://` to the beginning and `/new_post` to the end so it looks similar to `http://<your-web-server-domain>/new_post` and click **Add**
- 4. Copy the *Token* from your newly created webhook that appears under the *Existing outgoing webhooks* section
-
-3. **Set up your Mattermost slash command**
+2. **Set up your Mattermost slash command**
  1. Log in to your Mattermost account. Click the three dot menu at the top of the left-hand side and go to **Account Settings** > **Integrations** > **Slash Commands**
```
Comment (hanzei) [inline LINUX.md]: The correct Text should be 

> 1. Log in to your Mattermost account. Click the three dot menu at the top of the left-hand side and go to **Integrations** > **Outgoing Webhooks**

See #18

File: LINUX.md [hunk 4]
```diff
@@ -18,22 +18,16 @@ Here's how to start:
...
  2. Under *Add a new command*, enter `/gif` into **Command Trigger Word**
```
Comment (hanzei) [inline LINUX.md]: You might want to add an explanation about the displayname and the discription here. We could add a proposed one.

**Review by hanzei — CHANGES_REQUESTED** [review]

File: LINUX.md (hunk 2 above)
Comment (Lujeni) [reply LINUX.md]: :+1:

File: LINUX.md (hunk 3 above)
Comment (Lujeni) [reply LINUX.md]: :+1:

File: LINUX.md (hunk 2 above)
Comment (chakatz) [reply LINUX.md]: My version of Mattermost (3.5.0) does not have **Add** at the stage mentioned above, just at the very end you need to click **Save**.
Also, the Add Slash Command page explicitly states "Trigger word must be unique, and **cannot begin with a slash** or contain any spaces." It adds the slash on its own.
Probably best to have the port number in the example, people may think the code adds it automatically from the environment variable.

> similar to _http://<your.web.server.domain>:<MATTERMOST_GIPHY_PORT>/new_post_

**Comment by Lujeni** [comment]
@ultrabug @Hanzei done

**Pull request merged**